README.md

```diff
@@ -3,90 +3,134 @@ GOV.UK Frontend ·
 
-This file is needed to install a GOV.UK Frontend NPM package in your application. Most often it resides in the root of your application. You can create this file interactively using a `npm init` command.
+## Installation
+GOV.UK Frontend is currently in private beta. You will need to log in to NPM using credentials provided by the Design System team.
 
+Run the following command to log in:
+```
+npm login
+```
 
-## Install all components
+Enter the username, password and email address you were provided with, when prompted. 
 
+You can install all components or one or more individual components depending on your needs.
+
+To install all components, run:
 ```
 npm install --save @govuk-frontend/all
 ```
 
-## Install individual components
-
-Install a button component
 
+To install individual components (for example, a button), run:
 ```
 npm install --save @govuk-frontend/button
 ```
+You can find a list of all components in the [packages directory](https://github.com/alphagov/govuk-frontend/tree/master/packages). Each component has a README with installation and usage instructions.
 
-All components can be found in the packages directory.
-Each component has a README with installation and usage instructions.
+After you have installed GOV.UK Frontend the `@govuk-frontend` package will appear in your `node_modules` folder.
 
-## Quick start (using all components)
+## Import assets
 
-:rotating_light: **This will not work until packages are published to npm.** :rotating_light:
+Import styles into your main Sass file.
 
-Include the CSS and JavaScript
+To import all components, add the line:
+```CSS
+@import "@govuk-frontend/all/all";
+```
 
+To import an individual component (for example, a button), add the line:
+```CSS
+@import "@govuk-frontend/button/button";
 ```
+
+
+To resolve your `@import` declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) in Ruby.
+
+Below is a code sample you can add to your gulp configuration file using the gulp-sass package:
+```JS
+.pipe(sass({ includePaths: 'node_modules/' }))
+```
+
+To import images, configure your application to reference or copy the icons assets.
+
+Below is a code sample you can add to your [express.js](https://expressjs.com/) configuration:
+```JS
+app.use('/public', express.static(path.join(__dirname, '/node_modules/@govuk-frontend/icons')))
+```
+
+## Usage
+
+Copy and paste code from the examples in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/) to use GOV.UK Frontend in your service.
+
+<a id="option2"></a> 
+
+# Option 2: Add assets manually
+
+## Download assets
+
+Download the latest versions of the following assets and include them in your project:
+
+- [govuk-frontend/tree/master/dist/css](https://github.com/alphagov/govuk-frontend/tree/master/dist/css)
```

Comment:
Inconsistency here – icons doesn't list the full path like the others? (`icons` vs `govuk-frontend/tree/master/dist/icons`)? Personally, I'd prefer just `dist/css`, `dist/js`, `dist/icons`, but whatever we do we should be consistent.